plutus-chain-index/src/Plutus/ChainIndex/UtxoState.hs

```diff
@@ -185,7 +185,7 @@ rollbackWith
     -> Point
     -> UtxoIndex a
     -> Either RollbackFailed (RollbackResult a)
-rollbackWith _ PointAtGenesis (viewTip -> TipAtGenesis) = Right (RollbackResult TipAtGenesis mempty)
```

Comment:
I think to be completely correct this should be:
`rollbackWith f PointAtGenesis after = Right (RollbackResult TipAtGenesis (f mempty after))`

Reply:
Good catch!